packages/storybook/src/nimble/theme-provider/graphs.stories.ts

```diff
@@ -0,0 +1,29 @@
+import type { Meta } from '@storybook/html';
+import { getTokensStory, TokenName } from './tokens';
+
+const graphTokenNames = [
+    'graphGridlineColor',
+    'graphTrace1Color',
+    'graphTrace2Color',
+    'graphTrace3Color',
+    'graphTrace4Color',
+    'graphTrace5Color',
+    'graphTrace6Color',
+    'graphTrace7Color',
+    'graphTrace8Color'
+] as TokenName[];
```

Comment:
didn't dive deep but bet there is a way to avoid the `as` cast. please look into it